internal/pkg/service/stream/sink/tablesink/storage/repository/file_repo_delete_test.go

```diff
@@ -0,0 +1,228 @@
+		deleteEtcdLogs = etcdLogs.String()
+	}
+
+	// Check etcd logs
+	// -----------------------------------------------------------------------------------------------------------------
+	etcdlogger.Assert(t, `
+// Delete - AtomicOp - Read Phase
+➡️  TXN
+  ➡️  THEN:
+  001 ➡️  GET "storage/stats/target/123/456/my-source-1/my-sink-3/_sum"
+  002 ➡️  GET ["storage/stats/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/", "storage/stats/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z0")
+✔️  TXN | succeeded: true
+
+// Delete - AtomicOp - Write Phase
+➡️  TXN
+  ➡️  IF:
+  001 "storage/stats/target/123/456/my-source-1/my-sink-3/_sum" MOD EQUAL 0
+  002 ["storage/stats/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/", "storage/stats/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z0") MOD GREATER 0
+  003 ["storage/stats/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/", "storage/stats/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z0") MOD LESS %d
+  004 "storage/stats/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/my-volume-2/2000-01-01T02:00:00.000Z/value" MOD GREATER 0
+  005 "storage/stats/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/my-volume-3/2000-01-01T02:00:00.000Z/value" MOD GREATER 0
+  006 "storage/file/all/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z" VERSION NOT_EQUAL 0
+  ➡️  THEN:
+  001 ➡️  DEL "storage/file/all/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z"
+  002 ➡️  DEL "storage/file/level/local/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z"
+  003 ➡️  DEL "storage/file/level/staging/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z"
+  004 ➡️  DEL "storage/file/level/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z"
+  005 ➡️  DEL ["storage/slice/all/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/", "storage/slice/all/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z0")
+  006 ➡️  DEL ["storage/slice/level/local/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/", "storage/slice/level/local/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z0")
+  007 ➡️  DEL ["storage/slice/level/staging/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/", "storage/slice/level/staging/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z0")
+  008 ➡️  DEL ["storage/slice/level/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/", "storage/slice/level/target/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z0")
+  009 ➡️  DEL ["storage/stats/local/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/", "storage/stats/local/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z0")
+  010 ➡️  DEL ["storage/stats/staging/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z/", "storage/stats/staging/123/456/my-source-1/my-sink-3/2000-01-01T02:00:00.000Z0")
+  011 ➡️  PUT "storage/stats/target/123/456/my-source-1/my-sink-3/_sum"
```

Comment:
Save the new `_sum` - only for the `storage.TargetLevel` - only for the successfully imported data.